ui/overview_page.go

```diff
@@ -543,6 +553,29 @@ func (page *overviewPage) breakBalance(balance string) (b1, b2 string) {
 	return
 }
 
+func (page *overviewPage) updateTotransactionDetailsButtons() {
```

Comment:
You should use proper camel casing. "updateToTransactionDetailsButtons", not "updateTotransactionDetailsButtons".